qbdevice-ros: 4.0.2-1 in 'humble/distribution.yaml' [bloom]

GDqbrobotics opened the pull request:
Increasing version of package(s) in repository `qbdevice-ros` to `4.0.2-1`:

- upstream repository: https://bitbucket.org/qbrobotics/qbdevice-ros
- release repository: https://bitbucket.org/qbrobotics/qbdevice-ros2-release
- distro file: `humble/distribution.yaml`
- bloom version: `0.11.2`
- previous version for package: `null`

## qb_device

- No changes

## qb_device_bringup

- No changes

## qb_device_driver

- No changes

## qb_device_msgs

- No changes

## qb_device_ros2_control

- No changes

## qb_device_test_controllers

- No changes

GDqbrobotics committed:
qbdevice-ros: 4.0.2-1 in 'humble/distribution.yaml' [bloom]

clalancette commented on humble/distribution.yaml:
```diff
@@ -5839,6 +5839,20 @@ repositories:
       url: https://bitbucket.org/qbrobotics/qbshin-ros.git
       version: production-humble
     status: developed
+  qbdevice-ros:
```
Can you make this key `qb_device` like it was in Noetic?

clalancette reviewed (changes requested):
Besides the inline comment, it would also be nice to add in at least a `source` entry here.

GDqbrobotics committed:
Update distribution.yaml

Modified key qb_device like it was in Noetic and added source... I know I may sound crazy, but now the pull request is how it was supposed to be using bloom. I really don't get why there was no source and the name was taken from the repo. I will be more careful next time.

GDqbrobotics commented:
Should I open another pull request with bloom instead of editing this? I can't find any format error but I suppose there must be some...

clalancette commented:
> Should I open another pull request with bloom instead of editing this? I can't find any format error but I suppose there must be some...

There are trailing whitespaces on some of the lines you changed.

Besides that, it would be a good idea to rebase this onto the latest, as there have been some changes (unrelated to this PR) that you'll need in order to pass CI.
